Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1038,7 +1038,41 @@ - (void)_didCommitLayerTree:(const WebKit::RemoteLayerTreeTransaction&)layerTree
             _page->findClient().didAddLayerForFindOverlay(_page.get(), layer);
         }
     }
+
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+    [self _updateOverlayRegions];
```

Comment:
It would be nice to avoid this for commits where there were no changes to the layer tree, or the scrolling tree.

Reply:
Is it the case that changes to the layer tree always happen after the respective scrolling tree changes? I am wondering if it is sufficient to test for `layerTreeTransaction.changedLayerProperties().size()` and skip the update if the size is 0